readthedocs/doc_builder/environments.py

```diff
@@ -301,11 +301,35 @@ def run(self):
 
         self.start_time = datetime.utcnow()
         client = self.build_env.get_client()
+
+        # Create a copy of the environment to update PATH variable
+        environment = self._environment.copy()
+        # Default PATH variable
+        environment[
+            "PATH"
+        ] = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```

Comment:
Where did this come from? We should use something standard here, and document where we got it. Is this the Ubuntu default?

Reply:
Yes, this is the default PATH in Ubuntu. This is how I got it:

```
▶ docker run --user docs -it --rm readthedocs/build:ubuntu-22.04 /bin/bash
docs@bfe702e31cdd:~$ echo $PATH
/home/docs/.asdf/shims:/home/docs/.asdf/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
docs@bfe702e31cdd:~$
```

I should put that as a comment, probably 👍🏼 

> Are we explicitly not using the OS-level profile/rc files to get additional env information?

Yes. The way we are calling the commands inside Docker does not read these files.